be/src/storage/chunk_helper.cpp

```diff
@@ -261,7 +261,7 @@ std::vector<size_t> ChunkHelper::get_char_field_indexes(const vectorized::Schema
 
 void ChunkHelper::padding_char_columns(const std::vector<size_t>& char_column_indexes, const vectorized::Schema& schema,
                                        const starrocks::TabletSchema& tschema, vectorized::Chunk* chunk) {
-    size_t num_rows = chunk->num_rows();
+    uint32_t num_rows = static_cast<uint32_t>(chunk->num_rows());
```

Comment:
if new_offset is an std::vector<uint32_t>, new_offset.resize(size_t), the conversion error will throw?;